### ISSUE TITLE
There are a couple of questions

### ISSUE THREAD

**Issue by mqli0625** [git-disl/Fed-CDP#2]
I read your paper and found that this code is proved to be able to resist the gradient leakage attack in federated learning. I also saw two codes, one is to add differential privacy in federated learning, py file, using TensorFlow framework, and the other is to add gradient leakage, pyTorch framework, using Jupyter file.However, the two files cannot be run in series. Could you please provide the complete code of FedCDP to resist gradient leak attack? Can there be experimental results to prove that it can resist gradient leak attack?
